uk.co.compucorp.civicrm.hremails/email-templates/src/layouts/civihr-default.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
```

Comment:
what's this https://github.com/civicrm/civihr/blob/staging/uk.co.compucorp.civicrm.hremails/email-templates/src/layouts/default.html then? Could you check? I can't recall now

Reply:
I think this might be because some templates only existed in the 1.7 branch which I didn't have available. If there is already a default then it doesn't make sense to have civihr-default too so I'll see if I can use default.html instead.